package.json

```diff
@@ -18,17 +20,6 @@
   "author": "Apache Software Foundation",
   "license": "Apache-2.0",
   "bugs": "https://github.com/apache/cordova-test-platform/issues",
-  "homepage": "https://github.com/cordova/cordova-test-platform#readme",
```

Comment:
Is this meant to be removed? Description did not mention it.

Reply:
Looks like you are removing it instead from everywhere. So I approved it.

Thought it might have been better to point to the actual home page so people coming from npmjs has a quick link to our home page.

Reply:
I removed it since only four packages had this field set at all. Two of them to the default value